cli.js

```diff
@@ -43,7 +43,13 @@ Promise.longStackTraces();
 
 var conf = pkgConf.sync('ava');
 
-var pkgDir = path.dirname(pkgConf.filepath(conf));
+var pkgDir;
+
+try {
+	pkgDir = path.dirname(pkgConf.filepath(conf));
+} catch (err) {
```

Comment:
Check if `pkgConf.filepath(conf)` returns `null` instead. If it does, use `process.cwd()`, else call `path.dirname()` with the return value.